lib/ui/gallery_app_bar_widget.dart

```diff
@@ -364,11 +364,13 @@ class _GalleryAppBarWidgetState extends State<GalleryAppBarWidget> {
         message: showArchive ? "archive" : "unarchive",
         child: IconButton(
           icon: Icon(
-            Platform.isAndroid
+            Platform.isIOS
```

Comment:
Should it be Platform.isAndroid?

Reply:
Yes, my bad. Had added it for testing, will fix.